config/prod.json

```diff
@@ -10,6 +10,10 @@
   "drsHubUrlRoot": "https://drshub.dsde-prod.broadinstitute.org",
   "firecloudBucketRoot": "https://storage.googleapis.com/firecloud-alerts",
   "firecloudUrlRoot": "https://portal.firecloud.org",
+  "externalCreds": {
+    "urlRoot": "https://externalcreds.dsde-prod.broadinstitute.org",
+    "providers": []
+  },
```

Comment:
Another nit: let's sort the keys in alphabetical order here the same as in the other config files.